example/android/src/main/java/com/splendo/kaluga/example/keyboard/compose/KeyboardLayout.kt

```diff
@@ -71,14 +71,16 @@ class ComposeKeyboardActivity : AppCompatActivity() {
 fun KeyboardLayout() {
     MdcTheme {
         val focusHandler = LocalFocusManager.current
+        val focusRequester = remember { FocusRequester() }
         val viewModel =
             koinViewModel<KeyboardViewModel<ComposeFocusHandler>>(named(composeKeyboardViewModel)) {
                 parametersOf(
                     ComposeKeyboardManager.Builder(),
-                    ComposeFocusHandler(FocusRequester.Default),
                 )
             }
 
+        viewModel.editFieldFocusHandler.post(ComposeFocusHandler(focusRequester))
+
```

Comment:
I think posting from MainThread in layouts is not recommended. Can't you pass on injection creation ?

Reply:
I see, makes the impl a bit weird but I guess what we are doing is a bit weird to begin with. One more issue compose multiplatform will hopefully solve.

Reply:
Moved it to a launched effect just in case